### DISCUSSION TITLE
Kiali shows empty graph

### DISCUSSION THREAD

**Discussion by sterziev88** [kiali/kiali#7282] [Q&A]
### Describe the bug

Kiali does not show any traffic in UI for the namespace where are deployed my applications.
I am able to see my application in other panels like ( applications, workloads .. ) but there is nothing in traffic graph..
![image](https://github.com/kiali/kiali/assets/66157650/d8069ea3-34e4-4604-ae3f-b8b165597bd6)
![image](https://github.com/kiali/kiali/assets/66157650/e3b3abe6-bddb-40a6-87bc-914783ad06e1)
![image](https://github.com/kiali/kiali/assets/66157650/74af30fb-b6e0-48b0-91ae-a4a15e13c270)
![image](https://github.com/kiali/kiali/assets/66157650/73ac2611-2b22-4b4b-81eb-ed33eca27ade)

### Expected Behavior

To be able to see traffic for my applications

### What are the steps to reproduce this bug?

1. Install Istio with helm chart version 1.21
2. Install Kiali operator with helm chart version 1.81 

### Environment
*Learn about how to determine versions [here](https://kiali.io/docs/faq/general/#how-do-i-determine-what-version-i-am-running).*

- **Kiali version:*1.81*
- **Istio version:*1.21*
- **Kubernetes version:*1.29*

**Comment by jmazzitelli** [comment]
(_converting to Q&A Discussion as this is likely not a bug_)

This is a typical issue related to missing telemetry in the Istio backend (Prometheus). Have you consulted the FAQ on this? It provides details on how to figure out what is wrong:
* "Why is my graph empty?" : https://kiali.io/docs/faq/graph/#emptygraph
* "How do I inspect the underlying metrics used to generate the Kiali Graph?" https://kiali.io/docs/faq/graph/#queryprom

**Reply by SID21MAH** [reply]
What was fix for this, I am also having same issue where Kiali is showing traffic only for observability namespace where operator and Prometheus is installed but not showing traffic for other namespaces, I can see metrics in Prometheus for all namespaces.
Istio- 1.25.1
Kiali- 2.5

**Reply by jshaughn** [reply]
If you have metrics then typically the issue is that you have negligible traffic for the selected time period.
